pkg/scheduler/framework/interface.go

```diff
@@ -135,7 +135,8 @@ func NewPodsToActivate() *PodsToActivate {
 // code is not `Success`, the reasons should explain why.
 // NOTE: A nil Status is also considered as Success.
 type Status struct {
-	code    Code
+	code Code
+	// reasons is field that records the reason why it is not success.
```

Comment:
Hmm... I don't think it's that needed - L135 already explains it.

Reply:
IMO, L135 says that if code is non-success, you should explain why it is not successful on reasons field. 
But it doesn't mean that reasons field is only for failed reasons (not for success reasons). 

This is just my opinion, so please feel free to close this PR if you don't think it's necessary :)

Reply:
The way you are phrasing this also doesn't imply that reasons shouldn't be used for success.

I would modify L133-136 to add: When code is Success, all the other fields should be empty.

Reply:
Fixed, thanks @alculquicondor :)